lib/react_client.dart

```diff
@@ -77,7 +88,7 @@ class ReactDartComponentFactoryProxy extends ReactComponentFactoryProxy {
   JsObject call(Map props, [dynamic children]) {
     List reactParams = [
       generateExtendedJsProps(props, children),
```

Comment:
Actually, `children` should be converted beforehand and passed in here as well so the same result of any `Iterable`s is passed to both React JS and the Dart component. 

I'll update this.